tests/tests.c

```diff
@@ -720,7 +720,7 @@ bool sirtest_textstylesanity(void) {
     bool pass = si_init;
 
     printf("\t" WHITEB("--- explicitly invalid ---") "\n");
-    pass &= !sir_settextstyle(SIRL_INFO, 0xbbb, 800, 920);
+    pass &= !sir_settextstyle(SIRL_INFO, (sir_textattr)0xbbb, 800, 920);
```

Comment:
Why does sir_textattr need a cast? And shouldn't it be TA_CAST

Reply:
Yes to the latter, I'll fix that.

But why? Seems this is some Circle weirdness.  But since it's not wrong, I'll play along.  

Here's the error it throws without it:

```c
error: tests/tests.c:723
cannot convert prvalue int to sir_textattr
    pass &= !sir_settextstyle(SIRL_INFO, 0xbbb, 800, 920);
                                         ^
```

I actually really like some of the warnings and diagnostics that the Circle parser reports.  It doesn't have near as many warnings / diagnostics as Clang or GCC sadly.